<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchPageWidget</class>
 <widget class="QWidget" name="SwitchPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="saveConfigButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editPortDescButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="refreshAllPortInfoButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonFollowStyle</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="refreshMacTableInfoButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonFollowStyle</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="leftVerticalLayout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="portsTitleLabel">
         <property name="text">
          <string>Порты</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="portListTableView">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::EditKeyPressed</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="portInfoFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="portInfoLabel">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Порт xxx</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="closePortInfoButton">
              <property name="toolTip">
               <string>Закрыть</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resource.qrc">
                <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="portInetVlanDescLabel">
              <property name="text">
               <string>Internet Vlan:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="portIptvVlanDescLabel">
              <property name="text">
               <string>IPTV Unicast Vlan:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="multicastVlanDescLabel">
              <property name="text">
               <string>Multicast Vlan Member:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="portInetVlanValueLabel">
              <property name="text">
               <string>XXX</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="multicastVlanValueLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="portIptvVlanValueLabel">
              <property name="text">
               <string>XXX</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="rightVerticalLayout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="macTableTitleLabel">
         <property name="text">
          <string>Таблица MAC-адресов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="filterGroupBox">
         <property name="title">
          <string>Фильтрация</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="macRadioButton">
            <property name="text">
             <string>по MAC-адресу</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LineEdit" name="macLineEdit" native="true">
            <property name="inputMask" stdset="0">
             <string>hh-hh-hh-hh-hh-hh;_</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="portRadioButton">
            <property name="text">
             <string>по портам</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LineEdit" name="portListLineEdit" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="filterPortButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Отфильтровать</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="macAddressTableView">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="refreshPortInfoAction">
   <property name="text">
    <string>Обновить информацию по порту</string>
   </property>
   <property name="toolTip">
    <string>Обновить информацию по порту</string>
   </property>
  </action>
  <action name="refreshAllPortInfoAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/refresh.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Обновить информацию по портам</string>
   </property>
   <property name="toolTip">
    <string>Обновить информацию по портам</string>
   </property>
  </action>
  <action name="refreshMacTableInfoAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/refresh.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Обновить информацию по  MAC-адресам</string>
   </property>
   <property name="toolTip">
    <string>Обновить информацию по  MAC-адресам</string>
   </property>
  </action>
  <action name="showPortInfoAction">
   <property name="text">
    <string>Настройки порта</string>
   </property>
   <property name="toolTip">
    <string>Настройки порта</string>
   </property>
  </action>
  <action name="addPortToMulticastVlanAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/yes.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Добавить порт</string>
   </property>
   <property name="toolTip">
    <string>Добавить порт</string>
   </property>
  </action>
  <action name="removePortFromMulticastVlanAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/no.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Удалить порт</string>
   </property>
   <property name="toolTip">
    <string>Удалить порт</string>
   </property>
  </action>
  <action name="setPortInetServiceAction">
   <property name="text">
    <string>Интернет</string>
   </property>
   <property name="toolTip">
    <string>Интернет</string>
   </property>
  </action>
  <action name="setPortInetWithIptvStbServiceAction">
   <property name="text">
    <string>Интернет + Iptv Stb</string>
   </property>
   <property name="toolTip">
    <string>Интернет + Iptv Stb</string>
   </property>
  </action>
  <action name="saveConfigAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Сохранить изменения конфигурации</string>
   </property>
   <property name="toolTip">
    <string>Сохранить изменения конфигурации</string>
   </property>
  </action>
  <action name="editPortDescAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/edit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Изменить описание порта</string>
   </property>
   <property name="toolTip">
    <string>Изменить описание порта</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QWidget</extends>
   <header>lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
